profile/2018-03-01-hybrid.json

```diff
@@ -0,0 +1,163 @@
+{
+    "info":{
+        "name":"2018-03-01-profile",
+        "description":"Profile definition targeted for hybrid applications that could run on azure stack 1807 version and azure cloud"
+    },
+    "resource-manager": {
+        "microsoft.authorization": {
+			"2016-09-01": [
+				"locks"
+			],
+			"2015-07-01": [
```

Comment:
I don't see this API version here either: https://github.com/Azure/azure-rest-api-specs-pr/tree/master/specification/resources/resource-manager.

Reply:
@jianghaolu  Its here https://github.com/Azure/azure-rest-api-specs/tree/master/specification/resources/resource-manager/Microsoft.Authorization/stable/2016-09-01

Reply:
if you are looking roleassignments, they are here.
Microsoft.Azuthorization specs are in two different places :( (they are owned by two different teams)

https://github.com/Azure/azure-rest-api-specs/tree/master/specification/authorization/resource-manager/Microsoft.Authorization/stable/2015-07-01

Reply:
Ahh gotcha. Sorry about the confusion